src/components/anchor-navigation/anchor-navigation.test.tsx

```diff
@@ -0,0 +1,364 @@
+import React from "react";
+import { render, screen, act } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+
+import Textbox from "../textbox";
+import {
+  AnchorNavigation,
+  AnchorNavigationItem,
+  AnchorSectionDivider,
+} from ".";
+import { AnchorNavigationProps } from "./anchor-navigation.component";
+
+const SECTION_VISIBILITY_OFFSET = 200;
+
+const SELECTED_ITEM_STYLES = [
```

Comment:
**suggestion**: similar to @Parsium's comment, I feel we can remove this here and just assert the `backgroundColor` and `borderLeftColor` where appropriate

Reply:
Yeah I agree with this one (as already noted in reply to James's comment mentioning this), as contrary to the `SECTION_VISIBILITY_OFFSET` it should be fairly clear what these styles mean when they're used in a test